haskell/private/haddock_wrapper.sh.tpl

```diff
@@ -13,5 +14,5 @@ cleanup() { rm -rf "$TEMP"; }
 # XXX Override TMPDIR to prevent race conditions on certain platforms.
 # This is a workaround for
 # https://github.com/haskell/haddock/issues/894.
-TMPDIR=$TEMP %{haddock} "$@"
+TMPDIR=$TEMP "%{haddock}" "$@"
```

Comment:
This should be escaped in the template insertion code, via `shell.quote`:
https://github.com/bazelbuild/bazel-skylib/blob/0.6.0/lib/shell.bzl#L37-L49

Reply:
Done in 46f554b41439c87de16cf6f0f17929dfe0251a5e